test/Microsoft.AspNet.Mvc.WebApiCompatShimTest/ApiControllerTest.cs

```diff
@@ -419,7 +420,7 @@ public void ApiController_StatusCode()
             var result = controller.StatusCode(HttpStatusCode.ExpectationFailed);
```

Comment:
Also as @dougbu pointed out. That enum is used in functional tests. Those should be left alone since it's code making heavy use of `HttpClient`